packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -153,7 +154,7 @@ $Calendar-dayPicker-margin: 10px;
 }
 
 .day.dayIsHighlighted.dayIsFocused {
-  @include focus-border(1px, $ms-color-white);
+  @include focus-border(1px, WindowText);
```

Comment:
hmm. This seems like a hardcoded color that won't get replaced by theming. I'm not sure that's correct. Shouldn't this be scoped within a hc selector?

Reply:
it seems like we may be trading one bug for another here, careful